docs/main/kafka-knative-broker/README.md

```diff
@@ -0,0 +1,98 @@
+# Using Knative kafka broker
+If you want to use a Knative broker for communication between the different componenets (Data Index, Job Service and Workflows), you should use a reliable broker, i.e: not in-memory.
+
+Kafka perfectly fullfil this reliability need.
+
+## Pre-requisites
+
+1. A Kafka cluster running, see https://strimzi.io/quickstarts/ for a quickstart setup
+
+## Installation steps
+
+1. Configure and enable kafka broker feature in Knative: https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/
+  * i.e: 
+```console
+oc apply --filename https://github.com/knative-extensions/eventing-kafka-broker/releases/download/knative-v1.14.5/eventing-kafka-controller.yaml
+oc apply --filename https://github.com/knative-extensions/eventing-kafka-broker/releases/download/knative-v1.14.5/eventing-kafka-broker.yaml
+```
+    * Note the we are using the version `knative-v1.14.5` in this example, this may change, please refer to the offical documention link provided
+  * wait for the `kafka-broker-receiver` resource to be ready:
```

Comment:
s/`wait`/`Wait` to keep consistency with your style where the first word in the bullet point it capitalized.